prime-router/src/main/kotlin/cli/ValidateUtilities.kt

```diff
@@ -0,0 +1,61 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktError
+import com.github.ajalt.mordant.rendering.TextColors.green
+import com.github.ajalt.mordant.rendering.TextColors.red
+import gov.cdc.prime.router.config.validation.ConfigurationType
+import gov.cdc.prime.router.config.validation.ConfigurationValidationFailure
+import gov.cdc.prime.router.config.validation.ConfigurationValidationResult
+import gov.cdc.prime.router.config.validation.ConfigurationValidationService
+import gov.cdc.prime.router.config.validation.ConfigurationValidationServiceImpl
+import gov.cdc.prime.router.config.validation.ConfigurationValidationSuccess
+import java.io.File
+
+class ValidateUtilities() {
+    companion object ValidateUtilities {
+        val service: ConfigurationValidationService = ConfigurationValidationServiceImpl()
+
+        fun validateFiles(
```

Comment:
Is there any way to make calls to `ValidateYAMLCommand` since this code all seems pretty similar?

Maybe theres a way to run a command from a command otherwise you could directly call some of the functions.

Reply:
ah hadn't scrolled down enough. Looks like that was the impetus for this object.